src/gameplay/GameStart.js

```diff
@@ -32,16 +39,18 @@ function GameStart(props) {
   if(redirect){
     return (<Redirect to="/start" />);
   }
+
   function JoinGame(props) {
     if (props.isHost) {
       return (
         <button
-          type="button" className="btn btn-lg btn-primary" onClick={handleClick} style={{ marginBottom: 50 }}>
+          type="button" className="btn btn-lg btn-primary" onClick={handleClick} style={{ marginBottom: 50 }} disabled = {!isdisabled}>
```

Comment:
Naming Convention. Kind of Confusing. Either change `isDisabled` to `isEnabled` or change the logic so that you dont need to use the `!` operator